Content.Client/GameTicking/Managers/ClientGameTicker_RoundEndData_Container.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Client.RoundEnd;
+using Content.Shared.GameTicking;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Client.GameTicking.Managers
```

Comment:
file scope namespace

Reply:
hmmm so I'm not too sure what I need to do here? the namespace naming convention seems fine and synced with the actual source file destinations, and likewise with the other .cs files?
<img width="539" alt="image" src="https://github.com/space-wizards/space-station-14/assets/146375585/82868c9b-369f-42d8-af70-77b320a223b5">